src/compile/facet.ts

```diff
@@ -275,7 +268,10 @@ export class FacetModel extends Model {
       // axisGroup is a mapping to VgMarkGroup
       vals(this.component.axisGroup),
       flatten(vals(this.component.gridGroup)),
-      this.component.mark
+      extend(
+        this.component.mark[0],
```

Comment:
Why not make mark an object [here](https://github.com/vega/vega-lite/pull/1304/commits/d96190407525bbe130ca584dcb7c6959365d3999#r66190592) and just do `this.component.mark`?